cpp/src/io/orc/stripe_data.cu

```diff
@@ -1553,8 +1553,7 @@ extern "C" __global__ void __launch_bounds__(NTHREADS)
       __syncthreads();
       // Account for skipped values
       if (num_rowgroups > 0 && !s->is_string) {
-        uint32_t run_pos = s->top.data.index.run_pos[CI_DATA]
-                           << ((s->chunk.type_kind == BOOLEAN) ? 3 : 0);
+        uint32_t run_pos = s->top.data.index.run_pos[CI_DATA];
```

Comment:
It's really hard to understand the change from the diff. IIUC, run_pos has changed the meaning from the number of bool values to the number of bytes. But I see it used in lines like `vals_skipped = min(numvals, run_pos);` and can't understand how this use is still correct.

Reply:
Earlier, run_pos was in terms of bytes, but now as we are already updating it in `s->row_index_entry[2][ci_id] = (s->chunk.type_kind == BOOLEAN) ? (v << 3) + *cur : v;`, we don't have to shift here.